website/blog/2017-10-05-babel-turns-three.md

```diff
@@ -72,7 +72,7 @@ We've accomplished a lot, but we have to continue to work in multiple ways to ma
 
 ## Future Ideas
 
-- Providing a ["metadata" plugin](https://github.com/babel/notes/issues/34) for users to grep for patterns in their codebases. This will be useful to figure out how people are using certain proposals (similar to what was done in [tc39/proposal-optional-chaining#17](https://github.com/tc39/proposal-optional-chaining/issues/17) for coffescript regarding `?.`)
+- Providing a ["metadata" plugin](https://github.com/babel/notes/issues/34) for users to grep for patterns in their codebases. This will be useful to figure out how people are using certain proposals (similar to what was done in [tc39/proposal-optional-chaining#17](https://github.com/tc39/proposal-optional-chaining/issues/17) for coffeescript regarding `?.`)
```

Comment:
Since we're already touching it:

```suggestion
- Providing a ["metadata" plugin](https://github.com/babel/notes/issues/34) for users to grep for patterns in their codebases. This will be useful to figure out how people are using certain proposals (similar to what was done in [tc39/proposal-optional-chaining#17](https://github.com/tc39/proposal-optional-chaining/issues/17) for CoffeeScript regarding `?.`)
```